coforma/swift-tech-challenge
# Default image

# Default image fallback

# Summary
Incase the generation fails to create an image for an institution need a fallback image component
Related to #80 

## Type of change

Please delete options that are not relevant.

# How To Test
1. Scroll to an institution with no image
2. See the default
3. Click into the instution
4. See the default

# Checklist:
